src/common/profile-validator/profile-validator.service.ts

```
@@ -0,0 +1,203 @@
+      }
+    }
+
+    // If no format matches, return null or handle error
+    return null;
+  }
+
+  // Gets value tracing a path in a vc
+  private getAttributeValue(vc: any, pathValue: string): any {
+    if (vc.docFormat === 'json') {
+      return pathValue.split('.').reduce((acc, part) => {
+        return acc && acc[part];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)